src/admin/components/forms/field-types/Email/index.tsx

```diff
@@ -80,6 +82,10 @@ const Email: React.FC<Props> = (props) => {
         name={path}
         autoComplete={autoComplete}
       />
+      <FieldDescription
+        value={value || ''}
```

Comment:
Seems redundant to have to pass the value as empty string - here, and everywhere. Could be better pattern to handle within the component itself.